src/imp/linux_raw/conf/mod.rs

```diff
@@ -0,0 +1 @@
+pub(crate) mod auxv;
```

Comment:
I mean... why not replace `conf` feature with `auxv` feature?
I fell like conf is too general... I might be wrong

Reply:
This can be resolved

Reply:
It's true though; `conf` really is too general here. What would you think about the name "param"? That's also a little over-general, but in a way that might be less confusing?

Reply:
`sysinfo`?

Reply:
I'm hoping to avoid saying "sys", because these values describe a particular process; there may be different values among multiple processes on the same system.

Reply:
Hmm on who's terminology `sys` means 'values describe a particular process'?

Reply:
`machinfo`, Yes it might not sound the best, but I think it makes it clearer